pgjdbc/src/main/java/org/postgresql/util/PGPropertyUtil.java

```diff
@@ -1,125 +0,0 @@
-/*
- * Copyright (c) 2021, PostgreSQL Global Development Group
- * See the LICENSE file in the project root for more information.
- */
-
-package org.postgresql.util;
-
-import org.postgresql.PGProperty;
-
-import org.checkerframework.checker.nullness.qual.Nullable;
-
-import java.util.Locale;
-import java.util.Properties;
-import java.util.logging.Level;
-import java.util.logging.Logger;
-
-/**
- * routines to support PG properties
- */
-public class PGPropertyUtil {
```

Comment:
Class removal is a backward-incompatible change

Reply:
In broad terms, you are correct. However, this class was intended for internal use only. It is not a JDBC driver service provided for external clients.

I can restore this class (and optionally mar methods as deprecated).
What are your expectations here?

Reply:
The class was public, so people might be using it. We must maintain backward compatibility

Reply:
class is restored and marked as deprecated